theme/views.py

```diff
@@ -654,6 +657,11 @@ def email_verify(request, new_email, uidb36=None, token=None):
     if user is not None:
         user.email = new_email
         user.save()
+        try:
+            keycloak_id = KEYCLOAK_ADMIN.get_user_id(user.get_username())
+            KEYCLOAK_ADMIN.update_user(keycloak_id, payload={"email": new_email})
+        except Exception:
+            logging.exception("Failed to sync the email change to Keycloak")
```

Comment:
@devincowan - until we have logging based alerts, how do you feel about sending an email as well? This sync is not likely to fail but if it does fail then we need to know.

Reply:
I think email is probably our only option for now :(